Cannot render form in Firefox

Rendering any form fails with

```
Error loading form. Exception { message: "", result: 2153775105, name: "", filename: "http://localhost:5988/medic/_design…", lineNumber: 193110, columnNumber: 0, data: null, stack: "require<[418]</</getProcessor/<@htt…" }
```

That line number corresponds to this [XSLT service line](https://github.com/medic/medic-webapp/blob/master/static/js/services/xslt.js#L24):

```
processor.importStylesheet(response.data);
```

The current [medic-enketo-xslt](https://github.com/medic/enketo-xslt) build is [failing on travis](https://travis-ci.org/medic/enketo-xslt/jobs/216954087) with

```
./scripts/generate-browser-xml Firefox
[generate] Cleaning build/ dir...
[generate] Cleaned.
[generate] Generating XML with enketo-transformer reference implementation...
[generate] Transforming XML in browsers...
[generate] Generating XML in Firefox...
make: *** [generate] Error 1
```

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "font-awesome": "^4.7.0",
     "google-libphonenumber": "^2.0.14",
     "jquery": "^3.2.1",
-    "medic-enketo-xslt": "^1.0.0",
+    "medic-enketo-xslt": "^1.1.1",
     "messageformat": "^1.0.0",
     "moment": "^2.17.1",
     "nools": "^0.4.4",

